walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -18,6 +18,15 @@
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    //no attendees
+    if (events == null){
```

Comment:
is this somewhat on the right track?

Reply:
The code I wrote is only to make it through the first 2 test conditions. They didn't seem at all correct and obviously didn't work.

Reply:
As per the method signature, you need to return a list of TimeRange records: `Collection<TimeRange>`.  Return an int (0) probably wouldn't build I think